python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -11,11 +11,11 @@
 from cudf.core._compat import PANDAS_GE_120
 
 import dask_cudf
-from dask_cudf.groupby import _is_supported
+from dask_cudf.groupby import SUPPORTED_AGGS, _is_supported
 
 
-@pytest.mark.parametrize("aggregation", ["sum", "mean", "count", "min", "max"])
-def test_groupby_basic_aggs(aggregation):
+@pytest.mark.parametrize("aggregation", SUPPORTED_AGGS)
+def test_groupby_basic_frame(aggregation):
```

Comment:
I don't think I understand how this is a `frame` test now? The test looks the same as `test_groupby_basic_series`?

Reply:
The difference is pretty subtle; in `test_groupby_basic_series` we do the aggregation on one of the columns of the frame instead of the frame itself. Happy to make more explicit series groupby tests if needed

Reply:
Ah - I see now :)

It may make sense to combine the two tests and use `pytest.mark.parametrize` to cover both cases. However, this is not a blocker (I don't feel very strongly at all).

Reply:
Good point - I consolidated the tests into one based on a `series` parameter 🙂